server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -184,6 +188,11 @@ private boolean updateDto(DbSession dbSession, UpdateUser update, UserDto dto) {
     changed |= updateExternalIdentity(update, dto);
     changed |= updatePassword(update, dto, messages);
     changed |= updateScmAccounts(dbSession, update, dto, messages);
+    // TODO Add unit tests
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjl%2FSONAR-10248%2Fsecondary_emails_sync&issues=AWDHC7TeJC-F6PRGRIOl&open=AWDHC7TeJC-F6PRGRIOl)

[ISSUE_KEY]: AWDHC7TeJC-F6PRGRIOl